web/src/lib/components/utilities-page/duplicates/duplicates-compare-control.svelte

```diff
@@ -152,13 +152,30 @@
       <AssetViewer
         asset={$viewingAsset}
         showNavigation={assets.length > 1}
-        onNext={() => {
+        onNext={async () => {
```

Comment:
It would be cleaner to extract the body of those callbacks to dedicated method

Reply:
I agree.
Done.